src/dispatch/dispatcher.rs

```diff
@@ -74,11 +79,17 @@ impl<'a, 'b> Dispatcher<'a, 'b> {
     pub fn dispatch_par(&mut self, world: &World) {
         let stages = &mut self.stages;
 
-        self.thread_pool.install(move || {
-            for stage in stages {
-                stage.execute(world);
-            }
-        });
+        self.thread_pool
+            .read()
+            .unwrap()
+            .0
+            .as_ref()
+            .unwrap()
```

Comment:
There are  multiple places that have this kind of boilerplate. Could this be made helper function?

Reply:
I'll add a macro

Reply:
Rethinking about it I'm not convinced in use a macro or and helper function.
I don't see a real benefit in hide that part of code.

Also this is used only in two locations, and is difficult that will be used elsewhere.

So I prefer to leave it as is.